deployments/static/custom-driver.yaml

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nvidia-container-toolkit
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: nvidia-container-toolkit-daemonset
+  name: nvidia-container-toolkit-daemonset
+spec:
+  selector:
+    matchLabels:
+      app: nvidia-container-toolkit-daemonset
+  template:
+    metadata:
+      labels:
+        app: nvidia-container-toolkit-daemonset
+    spec:
+      tolerations:
+        - key: nvidia.com/gpu
+          operator: Exists
+          effect: NoSchedule
+      priorityClassName: system-node-critical
+      serviceAccountName: nvidia-container-toolkit
+      hostPID: true
+      containers:
+      - image: nvidia/container-toolkit:1.15.0-ubuntu20.04
+        command: ["/work/nvidia-toolkit"]
```

Comment:
Wondering why use `/work/nvidia-toolkit` instead of `/bin/bash -c` with below args ? Does the yaml from the operator requires this change or not ?

Reply:
This spec was something that I put together to test the deployment of the toolkit container on Kind -- independent of th GPU operator. I was not deploying the entrypoint config map in this case.

The GPU Operator may not need this change since it uses the entrypoint to set the driver envvars below. 

cc @cdesiniotis

Reply:
Gotcha. Thanks!